temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsImpl.java

```diff
@@ -159,6 +150,17 @@ public boolean awaitTermination(long timeout, TimeUnit unit) {
     return true;
   }
 
+  @Override
+  public void connect(@Nullable Duration timeout) {
+    channelManager.connect(HEALTH_CHECK_SERVICE_NAME, timeout);
```

Comment:
I think "get system info" can replace health check for this use case (and therefore you may want to change `disableHealthCheck` option to `disableEagerConnection`). I don't think there's value in doing both for workflow service.

Reply:
It will, yeah, I just didn't implement it yet. It will be in a separate PR. Don't want to mix concerns here.

Reply:
:+1: